doc/lua_api.txt

```diff
@@ -197,9 +197,9 @@ So the name should be `experimental:tnt`.
 Enforcement can be overridden by prefixing the name with `:`. This can
 be used for overriding the registrations of some other mod.
 
-Example: Any mod can redefine `experimental:tnt` by using the name
+Example: Any mod can redefine `experimental:time_machine` by using the name
```

Comment:
> To test if Travis handles skipping/building right?

Ahh... Okay. Sorry about that, at first I thought you might have been a spam user, and didn't check to see your history.